src/main/core/Subscriber/FunctionalEventSubscriber.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Claroline\CoreBundle\Subscriber;
+
+use Claroline\CoreBundle\Entity\Log\FunctionalLog;
+use Claroline\CoreBundle\Event\CatalogEvents\FunctionalEvents;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Contracts\EventDispatcher\Event;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class FunctionalEventSubscriber implements EventSubscriberInterface
```

Comment:
Should be `Claroline\CoreBundle\Subscriber\Log\FunctionalLogSubscriber`. Same for `SecurityEventSubscriber`.

Reply:
Done, thx :)